index.html

```diff
@@ -0,0 +1,193 @@
+            <h3 class="program-item-title">Lecture</h3>
+            <p class="program-item-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaera
+            </p>
+          </section>
+
+          <section class="program-item">
+            <div class="program-item-icon">
+              <img src="./img/exhibition_icon.png" alt="conference" />
+            </div>
+            <h3 class="program-item-title">Exhibition</h3>
+            <p class="program-item-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaera
+            </p>
+          </section>
+
+          <section class="program-item">
+            <div class="program-item-icon">
+              <img src="./img/workshop_icon.png" alt="conference" />
+            </div>
+            <h3 class="program-item-title">Workshop</h3>
+            <p class="program-item-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaera
+            </p>
+          </section>
+
+          <section class="program-item">
+            <div class="program-item-icon">
+              <img src="./img/show_icon.png" alt="conference" />
+            </div>
+            <h3 class="program-item-title">Show</h3>
+            <p class="program-item-text">
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaera
+            </p>
+          </section>
+        </div>
+
+        <button type="button" class="join-button">
+          Join React Global Summir 2022
+        </button>
+
+        <a class="program-link" href="#">See the whole program</a>
+      </section>
+
+      <section id="speakers" class="speakers-container">
+        <h2 class="speakers-title">Featured Speakers</h2>
+        <hr class="speakers-line" />
+        <div class="speakers-items-container">
+          <!-- This section is created dynamically in JavaScript -->
+        </div>
+
+        <button type="button" class="more-button">MORE</button>
+      </section>
+
+      <section id="partner" class="partner-container">
+        <h2 class="partner-title">Partner</h2>
+        <hr class="partner-line" />
+        <div class="partners">
+          <img class="partner-logo" src="./img/google_logo.png" alt="" />
+          <img class="partner-logo" src="./img/meta_logo.png" alt="" />
+          <img class="partner-logo" src="./img/spotify_logo.png" alt="" />
+          <img class="partner-logo" src="./img/microsoft_logo.png" alt="" />
+          <img class="partner-logo" src="./img/airbnb_logo.png" alt="" />
+        </div>
+      </section>
+
+    </main>
+    <footer class="footer-container">
+        <div class="footer-logo">
+          <img
+            src="./img/summit_logo.png"
+            alt="summit logo"
+          />
+        </div>
+        <div class="footer-info">
+          <p class="footer-text">2022 Modern React Lima.</p>
+          <p class="footer-text">Some Rights Reserved.</p>
+        </div>
+      </footer>
```

Comment:
Kindly make sure that your project sticks as much as possible to the template provided. 

- The position of the text and the elements are different from the template on the desktop version. Kindly fix the position of the elements and add the missing text. :pray: :pray:  You could use lorem ipsum text for the missing text.

Your design
![image](https://user-images.githubusercontent.com/98363075/203672884-dbc98d40-fc03-4b3a-a9e8-8fca54a9b79a.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/203672936-fe27b4e1-a9cf-4968-a088-a53d2b043cdc.png)